federation/pkg/kubefed/init/init.go

```diff
@@ -229,7 +230,25 @@ func initFederation(cmdOut io.Writer, config util.AdminConfig, cmd *cobra.Comman
 		return err
 	}
 
+	//to talk to federation ctrl plane
```

Comment:
Let's try to avoid arbitrary acronyms and vowel-omissions in the comments in the future. Comments are there to explain what is happening in the code. Let's not make it difficult to even read them.